doc/dftb+/manual/manual.bib

```diff
@@ -642,6 +642,18 @@ @article{dominguez15
 doi = {10.1021/acs.jpca.5b01732}
 }
 
+@article{onufriev2019
+author = {Onufriev, Alexey V. and Case, David A.},
+title = {Generalized Born Implicit Solvent Models for Biomolecules},
```

Comment:
Due to bibtex behaviour, needs the first letter protecting from lower case conversion: {B}orn